src/templateselector.cpp

```diff
@@ -498,6 +502,7 @@ void TemplateSelector::showInfo(QTreeWidgetItem *currentItem, QTreeWidgetItem *p
 			ui.rbCreateInFolder->setChecked(true);
 			ui.rbCreateInEditor->setEnabled(false);
 		} else {
+			ui.rbCreateInEditor->setChecked(true);

```

Comment:
This is probably not a good choice, as the user may want to set it in a folder. It should not be changed if the user switched to folder.

Reply:
ok, with the multifile icon it should be clear that a switch happened.